packages/pluggableWidgets/html-node-web/src/HTMLNode.xml

```diff
@@ -0,0 +1,326 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com.mendix.widget.web.htmlnode.HTMLNode" pluginWidget="true" needsEntityContext="true" offlineCapable="true" supportedPlatform="Web" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../node_modules/mendix/custom_widget.xsd">
+    <name>HTMLNode</name>
+    <description>Displays custom HTML</description>
+    <icon />
+    <properties>
+        <propertyGroup caption="General">
+            <propertyGroup caption="Tag">
+                <property key="tagName" type="enumeration" defaultValue="div" required="true">
+                    <!-- TODO: sudo make me a list-->
+                    <caption>Tag name</caption>
+                    <description />
+                    <enumerationValues>
+                        <enumerationValue key="div">div</enumerationValue>
+                        <enumerationValue key="span">span</enumerationValue>
+                        <enumerationValue key="p">p</enumerationValue>
+                        <enumerationValue key="ul">ul</enumerationValue>
```

Comment:
I was trying to find any reference on the items listed bellow `p` , but couldn't find them, do we need rest of the items?